filament/src/materials/skybox.mat

```diff
@@ -33,11 +34,33 @@ fragment {
 #define SHAPR_SKYBOX_TYPE_SOLID_COLOR 0
 #define SHAPR_SKYBOX_TYPE_GRADIENT 1
 #define SHAPR_SKYBOX_TYPE_ENVIRONMENT 2
+#define SHAPR_SKYBOX_TYPE_CHECKERBOARD 3
 
     float3 approxInverseSRGB(float3 srgb) {
         return pow(srgb, vec3(2.2));
     }
 
+    // Creates a checkerboard pattern of repeats fields in uv (must be in [0,1])
+    // Source: https://github.com/mattdesl/glsl-checker
+    float checker(float2 uv, float2 repeats) {
+        float cx = floor(repeats.x * uv.x);
+        float cy = floor(repeats.y * uv.y); 
+        float result = mod(cx + cy, 2.0);
+        return sign(result);
+    }
+
+    float3 shaprCheckerboardPattern(float2 positionScaledClipSpace) {
+        const float colorLight = 1.0;
+        const float colorDark = 0.53;
+
+        // Size of a square in pixels
+        const float2 squareSizePixels = float2(8.0, 8.0);
```

Comment:
Fixed checkerboard size would change pattern density when moving the app window between displays of different pixel density. I think the pattern density should be defined in the same coordinate system as UI elements are, (which adapts to such changes). So a scale factor, or the size in pixels should be an adjustable parameter. Can be done at a later step, but make sure we don't forget.

Reply:
I added a uniform to handle this case in https://github.com/shapr3d/filament/pull/10/commits/7637034148b0a529b342f4439bb7db7c4488a1ad

Reply:
How will this scale factor be actually set? Through a new method on `filament::Skybox`?

Reply:
Fixed in https://github.com/shapr3d/filament/pull/10/commits/b719aa86072d469d2c438e9047a55876e5fb04e3